extensions/functions_string.yaml

```diff
@@ -1048,6 +1048,35 @@ scalar_functions:
           char_set:
             values: [ UTF8, ASCII_ONLY ]
         return: "fixedchar<L1>"
+  -
+    name: initcap
+    description: >-
+      Converts the input string into init cap case. Capitalize the first character of each word in the
```

Comment:
init -> initial

Reply:
And cap -> capitalized

Reply:
Rewritten in 94ab9e7ba165bb2ae9a0a4c9c21e182eb57f85c1, was hard to make it sound good since there's no proper name for the initcap case.